Kernel/Thread.cpp

```diff
@@ -262,6 +262,12 @@ void Thread::exit(void* exit_value)
     set_should_die();
     u32 unlock_count;
     [[maybe_unused]] auto rc = unlock_process_if_locked(unlock_count);
+    if (m_thread_specific_range.has_value()) {
+        auto* region = process().space().find_region_from_range(m_thread_specific_range.value());
+        VERIFY(region);
+        if (!process().space().deallocate_region(*region))
+            dbgln("Failed to unmap TLS range, exiting thread anyway.");
```

Comment:
Should we include the error in the message?

Reply:
Sure, the best I could do is include the error _number_ though. Any ideas? :)

Reply:
Error was all I was hoping for.

Reply:
Oh, also, dealocate_region only returns a bool.